numpy/f2py/crackfortran.py

```diff
@@ -947,7 +959,17 @@ def analyzeline(m, case, line):
             block = 'python module'
         elif re.match(r'abstract\s*interface', block, re.I):
             block = 'abstract interface'
-        name, args, result, bind = _resolvenameargspattern(m.group('after'))
+        if block == 'type':
+            name, attributes, params = _resolvetypedefpattern(m.group('after'))
```

Comment:
Use:
```python
name, attributes, _ = _resolvetypedefpattern(m.group('after'))
```
as `params` is not currently used.

Reply:
I learn about this convention recently. Adopted !